man/man1/p11sak.1.in

```diff
@@ -513,6 +516,16 @@ A custom path for p11sak_defined_attrs.conf can be set with the environment vari
 P11SAK_DEFAULT_CONF_FILE. If none is set p11sak will first look for the file in the user 
 directory, followed by the standard installation path.
 .PP
+.SS "OCK_USER_PIN"
+The token PIN can be specified via the environment variable OCK_USER_PIN.
+If nothing is set and the option
+.B \-\-pin
+is not specified, p11sak will prompt for the token PIN interactively.
+.PP
+.SS "PKCS11_USER_PIN (deprecated)"
+Same as OCK_USER_PIN. If set, OCK_USER_PIN is prefered over PKCS11_USER_PIN.
+This environment variable is deprecated.
+.PP
```

Comment:
Please also add the PKCSLIB  variable to specify the PKCS 11 implementation while you are at it.

Reply:
Ok. But I will do it in separate commits.